data/rules.py

```diff
@@ -403,6 +403,56 @@
         'actions': '',
         'database': 'postgresql'
     },
+    {
```

Comment:
Can you also change the existing rules and tests to now be simpler? For example rule `remove_self_join`